src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -72,7 +98,7 @@ public CommandBase moveLift(DoubleSupplier speed) {
             elevatorBrake.set(false);
             isBraked = false;
         }, () -> {
-            elevatorMotor.set(speed.getAsDouble());
+            liftSpeed(speed.getAsDouble());
```

Comment:
If you're using `liftSpeed` here, you may not need the initialize and end functions to do anything other than set it to 0.